netlify.toml

```diff
@@ -9,6 +9,7 @@
 base = ""
 command = """
 pip install --upgrade pip \
+&& pip install src/integrations/* \
```

Comment:
We need to install integration packages to generate documentation from docstrings. I can update the primary build process once this is merged.